script.js

```diff
@@ -0,0 +1,9 @@
+
+alert("good afternoon, let's play a game. ");
+var startGame = confirm("are you ready?");
+if (startGame) { const randomNumber = Math.floor(Math.random() * 10) + 1;
```

Comment:
пишем тело ифа с новой строчки
```suggestion
if (startGame) { 
    const randomNumber = Math.floor(Math.random() * 10) + 1;
```